lms/djangoapps/grades/tests/test_new.py

```diff
@@ -329,7 +329,7 @@ class TestMultipleProblemTypesSubsectionScores(SharedModuleStoreTestCase):
     """
 
     SCORED_BLOCK_COUNT = 7
-    ACTUAL_TOTAL_POSSIBLE = 16.0   # solutions fork is using old version of DnD which does not count
+    ACTUAL_TOTAL_POSSIBLE = 17.0   
```

Comment:
@naeem91 You left some trailing whitespace on this line ^, which has broken the build. Can you please fix this?

Also please don't merge PRs that don't have a passing build ;)